routes/dao_coin_exchange.go

```diff
@@ -74,16 +82,115 @@ func (fes *APIServer) GetDAOCoinLimitOrders(ww http.ResponseWriter, req *http.Re
 		return
 	}
 
+	txnStatus := requestData.TxnStatus
+	if txnStatus == "" {
+		txnStatus = TxnStatusUnconfirmed
+	} else if txnStatus != TxnStatusUnconfirmed &&
+		txnStatus != TxnStatusConfirmed {
+
+		_AddBadRequestError(
+			ww,
+			fmt.Sprintf("GetDAOCoinLimitOrders: Invalid TxnStatus: %v. Options "+
+				"are {unconfirmed, confirmed}.", txnStatus),
+		)
+		return
+	}
+
 	utxoView, err := fes.backendServer.GetMempool().GetAugmentedUniversalView()
 	if err != nil {
 		_AddInternalServerError(ww, fmt.Sprintf("GetDAOCoinLimitOrders: Problem fetching utxoView: %v", err))
 		return
 	}
+	if txnStatus == TxnStatusConfirmed {
```

Comment:
Can we move the logic to get a utxo view to a separate helper function? Basically just take in a Boolean - something like `getUtxoView(uncommitted bool)` or maybe a different name. I would also recommend using the GetCommittedTipView function on blockchain since it's really clear what's happening, need to check if it's public tho.